src/lbaf/IO/configurationValidator.py

```diff
@@ -102,35 +146,49 @@ def validate(valid_schema: Schema, schema_to_validate: dict):
     def main(self):
         """ Main routine for the config validation. """
         # Validate skeleton
-        if self.is_valid(valid_schema=self.__skeleton, schema_to_validate=self.__config_to_validate):
+        if self.is_valid(
+            valid_schema=self.__skeleton,
+            schema_to_validate=self.__config_to_validate):
             self.__logger.info("Skeleton schema is valid")
         else:
             self.__logger.error("Skeleton schema is invalid")
-            self.validate(valid_schema=self.__skeleton, schema_to_validate=self.__config_to_validate)
+            self.validate(
+                valid_schema=self.__skeleton,
+                schema_to_validate=self.__config_to_validate)
 
         # Validate from_data/from_samplers
         if (from_data := self.__config_to_validate.get("from_data")) is not None:
             self.__logger.info("Reading from data was chosen.")
-            if self.is_valid(valid_schema=self.__from_data, schema_to_validate=from_data):
+            if self.is_valid(
+                valid_schema=self.__from_data,
+                schema_to_validate=from_data):
                 self.__logger.info("from_data schema is valid")
             else:
                 self.__logger.error("from_data schema is invalid")
-                self.validate(valid_schema=self.__from_data, schema_to_validate=from_data)
+                self.validate(
+                    valid_schema=self.__from_data,
+                    schema_to_validate=from_data)
         elif (from_samplers := self.__config_to_validate.get("from_samplers")) is not None:
             self.__logger.info("Simulate from samplers was chosen.")
-            if self.is_valid(valid_schema=self.__from_samplers, schema_to_validate=from_samplers):
+            if self.is_valid(
+                valid_schema=self.__from_samplers,
+                schema_to_validate=from_samplers):
                 self.__logger.info("from_samplers schema is valid")
             else:
                 self.__logger.error("from_samplers schema is invalid")
-                self.validate(valid_schema=self.__from_samplers, schema_to_validate=from_samplers)
+                self.validate(
+                    valid_schema=self.__from_samplers,
+                    schema_to_validate=from_samplers)
 
         # Validate algorithm
         if (algorithm := self.__config_to_validate.get("algorithm").get("name")) is not None:
             self.__logger.info(f"Checking algorithm schema of: {algorithm}")
-            if self.is_valid(valid_schema=self.__algorithm.get(algorithm),
-                             schema_to_validate=self.__config_to_validate.get("algorithm")):
+            if self.is_valid(
+                valid_schema=self.__algorithm.get(algorithm),
+                schema_to_validate=self.__config_to_validate.get("algorithm")):
```

Comment:
@ppebay PEP 8: E125 continuation line with same indent as next logical line
SAME AS LINE 151

Reply:
Should be fixed now